Cargo.toml

```diff
@@ -6,6 +6,7 @@ description = "ayb makes it easy to create, host, and share embedded databases l
 homepage = "https://github.com/marcua/ayb"
 documentation = "https://github.com/marcua/ayb#readme"
```

Comment:
Tests pass locally but fail in GitHub Actions
![image](https://github.com/marcua/ayb/assets/57725/fe11d93d-0d4b-402f-8e2b-d274976f1715)

Reply:
Building `nsjail` solved it!!! :)